spec/swagger_helper.rb

```diff
@@ -507,6 +507,1027 @@
                             type: "string",
                             format: "date",
                           },
+                          housing_cost_type: {
+                            type: :string,
+                            example: "rent",
+                            description: "required when category is 'rent_or_mortgage'",
+                          },
+                        },
+                      },
```

Comment:
@starswan It is the aftereffect of strict validations. We had tests sending `housing_cost_type` object which is not present in API documentation. Maybe its because of incorrect tests.

Reply:
Housing cost type is very strictly defined - it used to be like this (a comment) but is now strict so I'm really confused

Reply:
updated swagger docs to support "oneOf" for cash_transactions > outgoings